test/Dialect/FIRRTL/SFCTests/emit-omir.fir

```diff
@@ -1,4 +1,4 @@
-; RUN: firtool --repl-seq-mem --repl-seq-mem-file="dummy" --mem-to-reg-of-vec=0 --verify-diagnostics --verilog --emit-omir %s | FileCheck %s
```

Comment:
Nice catch re:dropping `verify-diagnostics`, kinda surprised it's not an error if there are none.